src/components/Addtask/index.jsx

```diff
@@ -0,0 +1,56 @@
+import { useState } from "react";
+import { useDispatch } from "react-redux";
+import PropTypes from "prop-types";
+import { addTask } from "/src/store/actions";
```

Comment:
@hoh-wtag for absolute import consider the following convention throughout the project

```suggestion
import { addTask } from "@store/actions";
```